logstash-core/api/lib/app/system/info_command.rb

```diff
@@ -4,20 +4,30 @@
 class LogStash::Api::SystemInfoCommand < LogStash::Api::Command
 
   def run
-    report = { "version"   => "1.0.0",
-               "host_name" => "foobar",
-               "plugins"   => { "count" => 10, "list" => [] }
+    plugins_list = available_plugins
+    {
+      "version"   => LOGSTASH_CORE_VERSION,
+      "hostname" => hostname,
+      "plugins"   => { "count" => plugins_list.count, "list" => plugins_list.sort }
     }
-    report
   end
```

Comment:
Do we want to return the logstash version on every call? I suspect the call on `/` should be enough no?